usagetracking/src/util/usagetracking/UsageTrackingService.java

```diff
@@ -545,17 +545,21 @@ public void setT( long t ) {
    }
 
 
-   public static class UsageTrackingData implements Serializable {
+   public static class UsageTrackingData implements Serializable, ExternalizableBean {
 
       private static final long serialVersionUID = -6848208128738327020L;
+
+
       /** timestamps */
+      @externalize(1)
```

Comment:
TLongList isn't directly externalizable, no?

Reply:
Let's switch this to TLongArrayList and all is well...